### DISCUSSION TITLE
Option B virker ikke (Mac M2)

### DISCUSSION THREAD

**Discussion by brimobrimo** [Eduap-com/WordMat#158] [Support]
Når man står på en ligning og trykker option B får man et vindue som siger vent jeg regner samt et vindue: "Kommandoen blev afbrudt efter 5s. Vil du prøve igen, og hvor lang tid vil du vente. (sekunder) Computeren låses i ventetiden".

**Comment by hazsdk** [comment]
Jeg har en elev med samme problem. Kan ikke helt gennemskue det.

Jeg kan dog skifte til geogebra som CAS og så virker det.
